RecoParticleFlow/PFTracking/src/PFDisplacedVertexCandidateFinder.cc

```diff
@@ -225,6 +235,8 @@ void PFDisplacedVertexCandidateFinder::link(const TrackBaseRef& el1,
     return;
   }
 
+  P = theMinimum_.crossingPoint();
```

Comment:
since this code is modified, perhaps update the name to stay with CMSSW naming style (no single-letter, no first-capital for locals)

Reply:
addressed in https://github.com/cms-sw/cmssw/pull/31121/commits/965eff78a97350337cf4e2f8cb640153834a8b10